### PR TITLE
Adapted scripts for Llama3.1 and fixed generative flags.

### PR DESCRIPTION
**Pull request by ShayekhBinIslam**
For Indonesian, accuracy is 0.6386934673366834 with chat-template (otherwise 0.5).

### CODE REVIEW

File: scripts/run_generative.py [hunk 1]
```diff
@@ -140,6 +140,8 @@ def main():
         is_prometheus = True
     elif "gemini" in args.model:
         model_modifier = "gemini"
+    elif "Llama-3.1" in args.model:
```
Comment (ljvmiranda921) [inline scripts/run_generative.py:143]: Is this the complete diff? How is the model modifier being handled? 🤔

**Comment by ljvmiranda921** [comment]
One way to debug if thie fix is working is to try it on a language that llama3.1 was trained upon and then compare it with the llama3 results.

We can try German, French, or Spanish? I dont think it was trained on Indonesian

**Comment by ShayekhBinIslam** [comment]
Sure! For `fra_Latn`, the result is `{"accuracy": 0.6541038525963149}`.

**Review by ljvmiranda921 — APPROVED** [review]

File: scripts/run_generative.py (hunk 1 above)
Comment (ShayekhBinIslam) [reply scripts/run_generative.py:143]: It was already there in main: https://github.com/for-ai/aya_rm_multilingual/blob/41a8292ba70152fe5d22ad9823ddf4997767b113/scripts/run_generative.py#L278

**Pull request merged**